src/topp/ConsensusID.cpp

```diff
@@ -792,10 +810,22 @@ class TOPPConsensusID :
           }
         }
       }
-      else
+      else // link spectra by RT and mz proximity and do ConsensusID on their PSMs
       {
+        if (in.size() != 1)
+        {
+          OPENMS_LOG_WARN << "ConsensusID on idXML without the --per_spectrum flag, expects a single idXML file."
```

Comment:
should this be a hard error?

Reply:
Yes maybe. I actually left it for "wrong" old workflows. But maybe they should better fail.